core/src/main/java/com/kin/ecosystem/core/Configuration.java

```diff
@@ -104,4 +105,15 @@ public static void setEnvironment(KinEnvironment environment) {
 
 	private Configuration() {
 	}
+
+	private static String getDeviceAcceptedLanguage() {
+		Locale defaultLocale = Locale.getDefault();
+		String acceptedLanguage;
+		if (TextUtils.isEmpty(defaultLocale.getCountry())) {
+			acceptedLanguage = defaultLocale.getLanguage();
```

Comment:
@YohayBar are we sure `defaultLocale.getLanguage()` always returns a 2 character code?